<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HardwareSelect</class>
 <widget class="QDialog" name="HardwareSelect">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hardware</string>
  </property>
  <widget class="QLabel" name="label2_108">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>12</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Serial</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_probe_usrp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>885</x>
     <y>40</y>
     <width>65</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Probe</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_hardware">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>40</y>
     <width>140</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Computer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USRP X3x0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USRP B2x0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HackRF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RTL2832U</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>802.11x Adapter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USRP B20xmini</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LimeSDR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bladeRF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Open Sniffer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PlutoSDR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USRP2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USRP N2xx</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bladeRF 2.0</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_guess">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>815</x>
     <y>40</y>
     <width>65</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Guess</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2_158">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>12</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Interface</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label2_mode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>60</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>TSI:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_serial">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>40</y>
     <width>170</width>
     <height>27</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_ip">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>130</width>
     <height>27</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2_17">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>12</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>IP Address</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame6_8">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>37</y>
     <width>950</width>
     <height>33</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label2_141">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>12</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Hardware</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_interface">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>170</width>
     <height>27</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>80</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>533</x>
     <y>80</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_apply_to_all">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>80</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply to All</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_daughterboard">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>40</y>
     <width>105</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CBX-120</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SBX-120</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UBX-160</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WBX-120</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TwinRX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XCVR2450</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DBSRX2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SBX-40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UBX-40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WBX-40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CBX-40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LFRX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LFTX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BasicRX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BasicTX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TVRX2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RFX400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RFX900</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RFX1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RFX1800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RFX2400</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label2_159">
   <property name="geometry">
    <rect>
     <x>695</x>
     <y>12</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Daughterboard</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label2_probe">
   <property name="geometry">
    <rect>
     <x>795</x>
     <y>80</y>
     <width>155</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Probing...Please Wait.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>frame6_8</zorder>
  <zorder>label2_108</zorder>
  <zorder>pushButton_probe_usrp</zorder>
  <zorder>comboBox_hardware</zorder>
  <zorder>pushButton_guess</zorder>
  <zorder>label2_158</zorder>
  <zorder>label2_mode</zorder>
  <zorder>textEdit_serial</zorder>
  <zorder>textEdit_ip</zorder>
  <zorder>label2_17</zorder>
  <zorder>label2_141</zorder>
  <zorder>textEdit_interface</zorder>
  <zorder>pushButton_ok</zorder>
  <zorder>pushButton_cancel</zorder>
  <zorder>pushButton_apply_to_all</zorder>
  <zorder>comboBox_daughterboard</zorder>
  <zorder>label2_159</zorder>
  <zorder>label2_probe</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
